docs/DeveloperGuide.md

```diff
@@ -168,19 +168,23 @@ The sequence diagram of the DeleteGradeCommand is shown below.
 
 ![DeleteGradeCommand.png](diagrams%2FDeleteGradeCommand.png)
 
-### Storage feature
+### Storage Load feature
 
-#### Implementation
+The `StudentStorage`, `GradeStorage` and `ComponentStorage` classes implement the feature to load data from the 
+data `.txt` files into their respective List objects at the start of the program.
 
-The `Storage` class is responsible for the automatic loading and saving of list data to and from `.txt` files,
-so that data will be retained between runs of the application.
+The load list methods for the Storage classes have largely similar logic flows.  
```

Comment:
Concise way of explaining the similar logic flows for the other classes, nice work!